src/libraries/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/EncryptedReference.cs

```diff
@@ -1,16 +1,17 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Diagnostics.CodeAnalysis;
 using System.Xml;
 
 namespace System.Security.Cryptography.Xml
 {
     public abstract class EncryptedReference
     {
-        private string _uri;
-        private string _referenceType;
-        private TransformChain _transformChain;
-        internal XmlElement _cachedXml;
+        private string? _uri;
```

Comment:
In what situation will this be null?

Reply:
It looks like it can't, so I have declared it as `private string _uri = null!;`. My other option was this:

![image](https://user-images.githubusercontent.com/263416/178093171-a6bf84df-36d0-4f0a-9577-dd2d0ce87b83.png)

But it still required `!`ing `_uri` from the getter

Reply:
I have a suspicion that maybe there's an attribute that I haven't thought of... So I'll leave this one as unresolved for the time being.